### PR TITLE
[TSD-2779], [TSD-2780]

### PR DESCRIPTION
**Pull request by DanMad**

### CODE REVIEW

File: lib/checkbox-set/checkbox-set.tsx [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+  error: string[];
+  success: string[];
+  value: string[];
+}
+
+export default class CheckboxSet extends React.Component<Props, State> {
+  static defaultProps: Partial<Props> = {
+    cards: false,
+    disabled: false,
+    onChange: () => {
```
Comment (victorliafook) [inline lib/checkbox-set/checkbox-set.tsx]: can you remove this? Or am I missing anything

File: lib/checkbox-set/checkbox-set.tsx [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+          {props.checkboxes.map((checkbox, index) => (
+            <GridItem key={index} modifiers="s-12 m-6 align-end">
+              <OptionalCard
+                disabled={props.disabled}
+                readOnly={props.readOnly}
+                visible={props.cards}>
+                <Checkbox
+                  id={checkbox.id}
+                  modifiers={
+                    state.value.indexOf(checkbox.value) !== -1
```
Comment (victorliafook) [inline lib/checkbox-set/checkbox-set.tsx]: it seems you forgot uglified code here

File: lib/checkbox-set/checkbox-set.tsx [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+                  checked={state.value.indexOf(checkbox.value) !== -1}
+                  onChange={handleChange}>
+                  {checkbox.label}
+                </Checkbox>
+              </OptionalCard>
+            </GridItem>
+          ))}
+        </Grid>
+        <div className={bem.getElement('border')} />
+        {(state.error.length !== 0 || state.success.length !== 0) &&
```
Comment (victorliafook) [inline lib/checkbox-set/checkbox-set.tsx]: can we set up a method?

File: lib/checkbox-set/checkbox-set.tsx (hunk 1 above)
Comment (DanMad) [reply lib/checkbox-set/checkbox-set.tsx]: This one we do need to leave, so we mitigate against browser errors in staging environments.

File: lib/checkbox-set/checkbox-set.tsx (hunk 2 above)
Comment (DanMad) [reply lib/checkbox-set/checkbox-set.tsx]: I fail you

**Pull request merged**